src/System.Private.ServiceModel/tests/Scenarios/SelfHostWcfService/WcfService.csproj

```diff
@@ -5,19 +5,19 @@
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
     <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
     <ProjectGuid>{CBE9909F-4CF7-4AD0-9307-BD0DFBB1105E}</ProjectGuid>
-    <OutputType>Exe</OutputType>
+    <OutputType>Library</OutputType>
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>WcfService</RootNamespace>
     <AssemblyName>WcfService</AssemblyName>
     <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
+    <OutputPath>..\Bridge\bin\WcfService\</OutputPath>
```

Comment:
See earlier comment.  I think we need our own wcf.targets at the root included by override.targets that sets where all tools and resources output artifacts will go via properties.  Example, set $(WcfToolsOutputPath) to some specific place in bin.  Set $(BridgeResourceFolder) at the same time so that TestProperties automatically shares where the resource artifacts are put when they are built.  This would also allow a build to place the resource folder entirely different, like a public share.